Debug App/Podfile

```diff
@@ -45,10 +45,14 @@ end
 def fix_linking(installer)
   
   installer.generated_projects.each do |project|
+    primer_target_names = project.targets.filter { |target|
+      !target.name.start_with?("PrimerSDK") && target.name.start_with?("Primer")
+    }.map { |t| t.name }
     project.targets.each do |target|
       if target.name == "PrimerSDK"
         target.build_configurations.each do |config|
-          config.build_settings['OTHER_LDFLAGS'] = '$(inherited) -framework "Primer3DS" -framework "PrimerKlarnaSDK"'
+          other_ldflags = "$(inherited) -framework #{primer_target_names.join(' -framework ')}"
+          config.build_settings['OTHER_LDFLAGS'] = other_ldflags
```

Comment:
Previously the string was: `$(inherited) -framework "Primer3DS" -framework "PrimerKlarnaSDK"`
Now it will be `$(inherited) -framework Primer3DS -framework PrimerKlarnaSDK`

The linker has no problem with the lack of quotes?

Reply:
It depends. Each token is essentially a command line argument for `ld` so it only needs to be quoted if it contains spaces. By convention, pods don't include spaces in their name. We could add it for a bit of future proofing though - wdyt?

Reply:
Seems it doesnt, I wonder would it be better to be safe, in the map: `.map { |t| "\"#{t.name}\"" }`